src/stirling/source_connectors/socket_tracer/conn_stats.h

```diff
@@ -75,6 +75,7 @@ class ConnStats {
     uint64_t conn_close = 0;
     uint64_t bytes_sent = 0;
     uint64_t bytes_recv = 0;
+    uint64_t cgid_curr = 0;
```

Comment:
Minor comment: My understanding is that this should never change. So it seems strange to label it as "current". I would prefer this to be just `cgid`.

Reply:
This can change if the process migrates cgroups. While this is not possible explicitly in kubernetes nothing stops someone from making this behavior materialize in an actual linux program, within kubernetes.

Reply:
Thanks for calling that out. I was vaguely aware of that, but didn't make the connection at first.